platform/comm/src/main/java/com/fmicodes/comm/DTO/RestaurantInfo.java

```diff
@@ -0,0 +1,90 @@
+package com.fmicodes.comm.DTO;
+
+public class RestaurantInfo {
+    private String name;
+    private String photoUrl;
+    private String location;
+    private double rating;
+    private Integer priceLevel;
+    private int userRatingsTotal;
+    private String openingHours;
+    private String googleMapsLink;
```

Comment:
change the primitive types with their corresponging objects, e.g. int -> Integer

Reply:
taken into account